src/pages/index.js

```diff
@@ -109,6 +211,11 @@ addCardButton.addEventListener("click", () => {
   newCardModal.open();
 });
 
+editAvatarButton.addEventListener("click", () => {
+  editAvatarModal.open();
+  formValidators[editAvatarModal._form.getAttribute("name")].resetValidation();
```

Comment:
`_form`

Please, delete the **low dash** (`underscore`) from methods if you want to use them `outside` a class because the `low dash` shows a **private** method which is used only inside the class.

Reply:
**Could be improved**


You can make method `getFormName` inside `ModalWithForm`


```js
  getFormName() {
    return this._form.getAttribute("name")
  }


```


And then call it here and in the other places as well


```js
addCardButton.addEventListener("click", () => {
  formValidators[newCardModal.getFormName()].resetValidation();  // for the other validators as well
  newCardModal.open();
});

editAvatarButton.addEventListener("click", () => {
  editAvatarModal.open();
  formValidators[editAvatarModal.getFormName()].resetValidation();
});

```